app/dao/mongo.go

```diff
@@ -0,0 +1,84 @@
+package dao
+
+import (
+	"fmt"
+
+	"github.com/companieshouse/payment-reconciler/app/config"
+	"github.com/companieshouse/payment-reconciler/app/models"
+	"github.com/globalsign/mgo"
+	"gopkg.in/mgo.v2/bson"
```

Comment:
Could we use `globalsign/mgo/bson` to keep it consistent across applications 
https://github.com/companieshouse/payments.api.ch.gov.uk/blob/develop/dao/mongo.go#L9

Reply:
Done 👍